### DISCUSSION TITLE
Media Library in its own window

### DISCUSSION THREAD

**Discussion by SteeveDumais** [OrchardCMS/OrchardCore#15857] [Q&A]
I am trying to have Media Library open up in its own tab I.e. in an iFrame in my application.  Without the left side menu bar, is this possible.  What I what my code to run is basically   initializeMediaApplication(true, '@Url.Action("MediaApplication", "Admin", new { area = "OrchardCore.Media" })');

**Comment by Skrypt** [comment]
Look at the example from the Wysiwyg editor when we open up the media library in a modal.
Media-HtmlBodyPart.Modal.cshtml

**Comment by SteeveDumais** [comment]
Is there a url to just have the media library without the admin menu.  The following URL goes to Media
https://localhost:44354/Admin/Media but the page has an admin menu.  If there is no URL without the admin menu, is there a way to turn off the admin menu for Media as I will never need it
